src/utils/attributes.js

```diff
@@ -0,0 +1,49 @@
+
+const CSSStyleMap = {
+    stroke: "stroke",
+    strokeWidth: "stroke-width",
+    strokeDasharray: "stroke-dasharray",
+    fill: "fill",
+    fillOpacity: "fill-opacity",
+    id: "id",
+    className: "class"
+}
+
+const defaultAttributes = {
+    stroke: "black",
+    strokeWidth: 1,
+    strokeDasharray: "",
```

Comment:
I think you could omit a default for `strokeDasharray`, as you omit falsey values later on.